igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -198,8 +198,7 @@ class JenkinsBuildMonitor implements PollingMonitor {
                                                         Project oldProject = new Project(name: project.name, lastBuild: build)
                                                         if (build.number != lastBuild
                                                                 || (build.number == lastBuild && cachedBuild.lastBuildBuilding != build.building)) {
-                                                            echoService.postEvent(
-                                                                    new BuildEvent(content: new BuildContent(project: oldProject, master: master)))
+                                                            postEvent(echoService, cachedBuild, oldProject, master)
```

Comment:
Yes, it would trigger subsequent builds if the redis is lost ... I figured it unlikely someone would add a new job and setup a trigger before we indexed the job.

One could argue that the impact of a lost redis is more severe than failing to trigger on the first build of a brand new job ... even if it's something that seldom occurs.

I don't even know if we need to check indexed builds per master ... might be able to get away with just checking that anything has been indexed (ie. `cachedBuilds.isEmpty()`) ... I'll play around with that.